superchain/configs/mainnet/superlumio.yaml

```diff
@@ -0,0 +1,23 @@
+name: SuperLumio
+chain_id: 8866
+public_rpc: https://mainnet.lumio.io
+sequencer_rpc: https://mainnet.lumio.io
+explorer: https://explorer.lumio.io
+
+superchain_level: 2
+superchain_time: null # Missing hardfork times are NOT yet inherited from superchain.yaml
+
+batch_inbox_addr: "0xa12cf34001e553dc254d131105364351f5174d75"
+
+genesis:
+  l1:
+    hash: "0xbe51a75131cf63e18596329bd4f8b805ce8dd6a0b17787d5da10f563616f1f1e"
+    number: 19314558
+  l2:
+    hash: "0x01711dc53a82967a34f4b38f7dff3ae8fa760882bb1e2ace50bb01e9ff420a9b"
+    number: 0
+  l2_time: 1708984631
+
+canyon_time: 0
+delta_time:  0
+ecotone_time:  1710374401 # Thu 14 Mar 2024 00:00:01 UTC
```

Comment:
The `canyon_ tim`e doesn't look correct to me, as evidenced by querying the RPC endpoint: 
```json
➜  ~ curl https://mainnet.lumio.io \
  -X POST \
  -H "Content-Type: application/json" \
  --data '{"method":"eth_getBlockByNumber","params":["0x0",false],"id":1,"jsonrpc":"2.0"}' | jq ".result"
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  1525  100  1446  100    79   2608    142 --:--:-- --:--:-- --:--:--  2747
{
  "baseFeePerGas": "0x3b9aca00",
  "difficulty": "0x0",
  "extraData": "0x424544524f434b",
  "gasLimit": "0x1c9c380",
  "gasUsed": "0x0",
  "hash": "0x01711dc53a82967a34f4b38f7dff3ae8fa760882bb1e2ace50bb01e9ff420a9b",
  "logsBloom": "0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
  "miner": "0x4200000000000000000000000000000000000011",
  "mixHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
  "nonce": "0x0000000000000000",
  "number": "0x0",
  "parentHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
  "receiptsRoot": "0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421",
  "sha3Uncles": "0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347",
  "size": "0x209",
  "stateRoot": "0xc3feec976dd5ff2a420a13df675e8d4688257204bb5f5cee42b7797da4631f58",
  "timestamp": "0x65dd0937",
  "totalDifficulty": "0x0",
  "transactions": [],
  "transactionsRoot": "0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421",
  "uncles": []
}
➜  ~ curl https://mainnet.lumio.io \
  -X POST \
  -H "Content-Type: application/json" \
  --data '{"method":"eth_getBlockByNumber","params":["0x1",false],"id":1,"jsonrpc":"2.0"}' | jq ".result"
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  1686  100  1607  100    79   3153    155 --:--:-- --:--:-- --:--:--  3312
{
  "baseFeePerGas": "0x3b5dc100",
  "difficulty": "0x0",
  "extraData": "0x",
  "gasLimit": "0x1c9c380",
  "gasUsed": "0xfa01",
  "hash": "0xb09e4f3cb812783db2394bc83edc71b155b99da105d82917b89d06720764b068",
  "logsBloom": "0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
  "miner": "0x4200000000000000000000000000000000000011",
  "mixHash": "0x8f92c4b30f9fc5f58aabb4e84a0bf8311d4bfbd46de011065ef2792352c2ec7d",
  "nonce": "0x0000000000000000",
  "number": "0x1",
  "parentHash": "0x01711dc53a82967a34f4b38f7dff3ae8fa760882bb1e2ace50bb01e9ff420a9b",
  "receiptsRoot": "0x7710dbaebd9b8d8b9a5ea06c21d6bdbfdf300950b4c03530d972262f6151c798",
  "sha3Uncles": "0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347",
  "size": "0x388",
  "stateRoot": "0xcc87cc9399785389f52e9d4acb6dfe93623f542e2ccbf46e349deb4e9f16966e",
  "timestamp": "0x65dd0939",
  "totalDifficulty": "0x0",
  "transactions": [
    "0xf2d5af2baec168557c6779309d010f44dc849b6154ebb161be2aae92486e7dbe"
  ],
  "transactionsRoot": "0xd316da35b03e9607eff4de703d10e349a35556c6328e071d5ba6bd786799d561",
  "uncles": [],
  "withdrawals": [],
  "withdrawalsRoot": "0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421"
}
➜  ~ curl https://mainnet.lumio.io \
  -X POST \
  -H "Content-Type: application/json" \
  --data '{"method":"eth_getBlockByNumber","params":["0x206767",false],"id":1,"jsonrpc":"2.0"}' | jq
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  1825  100  1741  100    84   3522    169 --:--:-- --:--:-- --:--:--  3686
{
  "jsonrpc": "2.0",
  "result": {
    "baseFeePerGas": "0xfc",
    "blobGasUsed": "0x0",
    "difficulty": "0x0",
    "excessBlobGas": "0x0",
    "extraData": "0x",
    "gasLimit": "0x1c9c380",
    "gasUsed": "0xab27",
    "hash": "0x6ac812fb41d2464bc467331434c40363e9f357594c2d0e0a7165d92f2f52dd5d",
    "logsBloom": "0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
    "miner": "0x4200000000000000000000000000000000000011",
    "mixHash": "0x95cd7d4bcd7dac72d4f9ac6d79f2d1d119a5d7c20fc1ebd45171b27b9202fefa",
    "nonce": "0x0000000000000000",
    "number": "0x206767",
    "parentBeaconBlockRoot": "0xb9c0037ccbecde07bd807924a41059398fde4a9a62ab1355cc5d5c2124103d88",
    "parentHash": "0xdbbc5285e941917add666ebf64ccd0e90fd075fbb408faad81663cab8f293cd3",
    "receiptsRoot": "0xde76d6249aaee13c1d4e5590125ab25c27423aff9a7b5bd525ff0572213828dc",
    "sha3Uncles": "0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347",
    "size": "0x347",
    "stateRoot": "0x3db733ccc20afced46eedb000f718e4f31c7c86babba543cd495105150dd9d41",
    "timestamp": "0x661dd805",
    "totalDifficulty": "0x0",
    "transactions": [
      "0xc98cfaf2a3376791644908543898cea76a0cb7f1bff01bd53b53fc9f6bd93ca0"
    ],
    "transactionsRoot": "0x86f625d6a51dcdcd2b8dbdb0cebf8b2921df3807fac9eeb92ad0b4d4928d4296",
    "uncles": [],
    "withdrawals": [],
    "withdrawalsRoot": "0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421"
  },
```

We can see that at block 0 on this chain, there is no withdrawals field. This implies shanghai was not active, and therefore canyon was not active.